docs/docs/how-it-works/existing-game.md

```diff
@@ -0,0 +1,42 @@
+---
+sidebar_position: 100
+---
+
+# Porting Existing Game
+
+Amazing that you're considering adapting your game for multiplayer on Rune! 🥳
+
+We recommend you follow **approach 1** below, which will take care of any boilerplate and provide a nice development experience with Rune-specific ESLint & Vite plugins. You can quickly get a minimal multiplayer version of your game working, then add more game logic step-by-step. Alternatively, you can add the SDK manually.
+
+### Approach 1: Rune Template
+
+Create a new Rune game project by running:
+
+```sh
+npx rune-games-cli@latest create
+```
+
+You now have a simple example game with game logic and client rendering files. You can then copy your game logic and rendering code into the `logic.ts` and `client.ts` files respectively. Take a look at [Quick Start](../quick-start) for an introduction to these files or [Syncing Game State](./syncing-game-state) for a more in-depth explanation.
+
+We recommend that you enable ESLint in your editor to detect issues while developing your game. The template code already has the necessary configuration for ESLint.
+
+### Approach 2: Manual Setup
+
+This approach requires you to be careful, especially if you're using TypeScript as you will need to update your TypeScript setup to export a `logic.js` file. If you still like to do the setup manually, here's how you do it:
+
+1. Create a file called `logic.js` that has all [game logic](../quick-start#game-logic) and calls `Rune.initLogic()`
+2. Create a file called `client.js` that's responsible for [rendering](../quick-start#rendering) and calls `Rune.initClient()`
+3. Load the SDK and the two files mentioned above in your `index.html` in the following order:
```

Comment:
This script must be loaded before any other script.

Reply:
Done in https://github.com/rune/rune-multiplayer-web-games/pull/318/commits/999e372252e8eac073cb3ffc9ae8ec3cf43a0679